example/advection/advection_package.cpp

```diff
@@ -74,6 +74,9 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   Real ang_2_vert = pin->GetOrAddBoolean("Advection", "ang_2_vert", false);
   Real ang_3_vert = pin->GetOrAddBoolean("Advection", "ang_3_vert", false);
 
+  auto fill_derived = pin->GetOrAddBoolean("Advection", "fill_derived", true);
+  pkg->AddParam<>("fill_derived", fill_derived);
```

Comment:
I don't think we need the `<>`, do we? (Also in lots of other places)

Reply:
Ah probably. I just copied convention.